### PR TITLE
Doc page NumberInput component

### PR DESCRIPTION
**Pull request by raquelarrojo**

### CODE REVIEW

File: website/screens/components/number-input/usage/NumberInputUsagePage.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { DxcStack, DxcText, DxcList } from "@dxc-technology/halstack-react";
+import HeadingLink from "../../../common/HeadingLink";
+import DocFooter from "../../../common/DocFooter";
+
+const NumberInputUsage = () => {
```
Comment (marcialfps) [inline website/screens/components/number-input/usage/NumberInputUsagePage.tsx:5]: Please, call it `NumberInputUsagePage` to have all the same naming.

File: website/screens/components/number-input/specs/NumberInputSpecsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            alt="Design specifications of the number input component"
+          />
+        </Figure>
+        <DxcText as="p">
+          The number input <Code>color</Code>, <Code>typography</Code>,{" "}
+          <Code>border</Code>, <Code>spacing</Code>, <Code>width</Code> and{" "}
+          <Code>margin</Code> specifications are inherited from the text input,
+          for reference{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/text-input"
```
Comment (marcialfps) [inline website/screens/components/number-input/specs/NumberInputSpecsPage.tsx:34]: I think this should be a link to `/components/text-input/specifications`.

File: website/screens/components/number-input/specs/NumberInputSpecsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <DxcText>Spin button decrease</DxcText>
+          <DxcText>Error indicator</DxcText>
+          <DxcText>Error message</DxcText>
+          <DxcText>Value</DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Accessibility</HeadingLink>
+        <HeadingLink level={4}>WAI-ARIA</HeadingLink>
+        <DxcStack>
```
Comment (marcialfps) [inline website/screens/components/number-input/specs/NumberInputSpecsPage.tsx:86]: Change to `DxcList`.

File: website/screens/components/number-input/specs/NumberInputSpecsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+          <DxcText>
+            WAI-ARIA Authoring practices 1.2 -{" "}
+            <DxcLink
```
Comment (marcialfps) [inline website/screens/components/number-input/specs/NumberInputSpecsPage.tsx:89]: I am using `newWindow` prop to open external links in other tab.

**Review by marcialfps — CHANGES_REQUESTED** [review]

File: website/pages/components/number-input/specifications.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import Head from "next/head";
+import type { ReactElement } from "react";
+import NumberInputSpecsPage from "../../../screens/components/number-input/specs/NumberInputSpecsPage";
+import NumberInputPageLayout from "../../../screens/components/number-input/NumberInputPageLayout";
+
+const Specifications = () => {
+  return (
+    <>
+      <Head>
+        <title>Number Input — Halstack Design System</title>
```
[the remark this answers is not in the document]
Comment (raquelarrojo) [reply website/pages/components/number-input/specifications.tsx]: Done.

File: website/screens/components/number-input/usage/NumberInputUsagePage.tsx (hunk 1 above)
Comment (raquelarrojo) [reply website/screens/components/number-input/usage/NumberInputUsagePage.tsx]: Done. The button component is not following this naming either. Please change it in the [PR](https://github.com/dxc-technology/halstack-style-guide/pull/729).

File: website/screens/components/number-input/specs/NumberInputSpecsPage.tsx (hunk 2 above)
Comment (raquelarrojo) [reply website/screens/components/number-input/specs/NumberInputSpecsPage.tsx]: Done.

File: website/screens/components/number-input/specs/NumberInputSpecsPage.tsx (hunk 3 above)
Comment (raquelarrojo) [reply website/screens/components/number-input/specs/NumberInputSpecsPage.tsx]: Done.

File: website/screens/components/number-input/specs/NumberInputSpecsPage.tsx (hunk 4 above)
Comment (raquelarrojo) [reply website/screens/components/number-input/specs/NumberInputSpecsPage.tsx:90]: Done.

File: website/screens/components/number-input/specs/NumberInputSpecsPage.tsx [hunk 6]
```diff
@@ -31,8 +31,9 @@ const NumberInputSpecs = () => {
           <Code>margin</Code> specifications are inherited from the text input,
           for reference{" "}
           <DxcLink
-            href="https://developer.dxc.com/design/guidelines/components/text-input"
+            href="https://developer.dxc.com/design/guidelines/components/text-input/specifications"
```
Comment (marcialfps) [inline website/screens/components/number-input/specs/NumberInputSpecsPage.tsx:34]: I was referring to this site, not to developer central.

**Pull request merged**